package.json

```diff
@@ -1,7 +1,38 @@
 {
+  "license": "MIT",
   "dependencies": {
     "form-data": "^2.1.2",
     "node-fetch": "^1.6.3",
-    "syncano-server": "^0.7.2-43"
+    "syncano-server": "beta"
+  },
+  "scripts": {
+    "test": "yarn && npm-run-all test:lint test:e2e:coverage",
+    "test:e2e-single": "cross-env NODE_ENV=test mocha -s 1800 --reporter spec --require babel-register --timeout 4000",
+    "test:e2e-single:coverage": "nyc --produce-source-map=true cross-env NODE_ENV=test mocha -s 1800 --reporter spec --require babel-register --timeout 4000",
+    "test:e2e": "npm run test:e2e-single 'test/*.test.js'",
+    "test:e2e:coverage": "nyc --produce-source-map=true npm run test:e2e",
+    "test:lint": "standard --fix"
```

Comment:
Let's use lint-staged and husky in all projects https://github.com/Syncano/syncano-server-js/blob/devel/package.json Here I use standard and prettier which together give best result. 

Also take a look at:
```
"test": "npm run test:unit && npm run test:e2e",
"test:base": "cross-env NODE_ENV=test nyc mocha --reporter spec --require babel-register",
"test:unit": "npm run test:base -- test/unit/**/*.js",
"test:e2e": "npm run test:base -- --timeout 30000 --slow 8000 test/e2e/*.js",
"test:coverage": "nyc report --reporter=text-lcov > coverage.lcov && codecov",
"test:watch": "mocha --watch test/**/*.js --compilers js:babel-register",
```

I found that structure realiable and nice looking, expect `test:watch` - it can be replaced with `npm run test:unit --watch` or `npm run test:e2e --watch`